src/runtimes/node/utils/included_files.ts

```diff
@@ -53,9 +53,13 @@ export const getPathsOfIncludedFiles = async (
     cwd: basePath,
     dot: true,
     ignore: excludePatterns,
-    followSymbolicLinks: true,
-    throwErrorOnBrokenSymbolicLink: true,
+    onlyFiles: false,
+    // get directories as well to get symlinked directories,
```

Comment:
😅 This is what I meant. Each property getting its own comment.

```suggestion
    // get directories as well to get symlinked directories
    onlyFiles: false,
```